sunpy/time/time.py

```diff
@@ -142,67 +137,64 @@ def convert_time(time_string, **kwargs):
 
 @convert_time.register(pandas.Timestamp)
 def convert_time_pandasTimestamp(time_string, **kwargs):
-    return time_string.to_pydatetime()
+    return ap.Time(time_string)
```

Comment:
I am impressed this works.

Reply:
I was too. `pandas.Timestamp` inherits `datetime`. So `isinstance` checks out. And it works.

Reply:
Just to be safe, maybe we should do `ap.Time(time_string.to_pydatetime())`.